db/migrate/20200501123428_create_goals.rb

```diff
@@ -0,0 +1,15 @@
+class CreateGoals < ActiveRecord::Migration[6.0]
+  def change
+    create_table :goals do |t|
+      t.string :name
+
+      t.timestamps
+    end
+
+    Goal.create(name: 'Pfadi (er) leben')
```

Comment:
the problem we faced often when doing this is, that a migration is always in place whereas the model might be removed. when running all migrations in a certain state, where the model isn't there, you will have an error because the model is not found. Also more [here](https://medium.com/@schmijos/execute-sql-in-rails-migrations-cdb26f51c683). so i always require to change that in my code reviews to use raw SQL:

```rb
    execute "INSERT INTO goals VALUES('Pfadi (er) leben', 'Diversität und Inklusion', 'Umwelt', 'Prävention', 'Dimension und Austausch')"
```